3rdparty/openexr/IlmThreadConfig.h.cmakein

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) Contributors to the OpenEXR Project.
+
+// This file is auto-generated by the cmake configure step
+
+#ifndef INCLUDED_ILMTHREAD_CONFIG_H
+#define INCLUDED_ILMTHREAD_CONFIG_H 1
+
+#pragma once
+
+//
+//
+// C++ namespace configuration / options
+
+#cmakedefine01 ILMTHREAD_THREADING_ENABLED
```

Comment:
I guess this is where it would be possible to make `ILMTHREAD_THREADING_ENABLED` default to `1`.

Reply:
Done (although not right here).